webapp/service/api/cancelableRequest.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios'
+
+export const cancelableRequest = ({ method = 'get', url, config = {} }) => {
```

Comment:
I wouldn't pass the entire config object but only the "params"

Reply:
params is only used by get, data is used by post.
if you want to pass only params, then we need a canelableGetRequest that calls the generic cancelableRequest

Reply:
can make the interface homogeneous among get and post, pass only params and take care of how the config object is created only inside the cancelableRequest function? it would simplify a bit the usage of it.

Reply:
to me an homogeneous interface looks nicer and if for some reason you have to switch from a get to a post it would be easier.
Passing the entire "config" object makes it dependent on the implementation (axios) and it's not true that developers are more used in using different ways to send a get or a post; consider for example the JQuery shorthands $.get and $.post, they both take a "data" object as parameter and they are just shorthands for $.ajax({url, type, data, ...})